### PR TITLE
#17 Added a Xerces-c toolchain compilation.

### LINKED ISSUE #17: Xerces recent version causes problem with c++98 compilation
Version 3.2.2 embedded on Fedora 29 makes libxml compilation fails if we force to compile with -std=c++98 flag on recent compiler (gcc 8.2) because Xerces assumes that CSTDINT and ICU are avalaible which is not if we force old standard.
One simple solution is to provide a toolchain to compile Xerces ourselves and not use the system package. In Xerces 3.2.2 the CMake compilation module is not enough parametrizable for our purpose so for the moment we will use the autoconf compilation procedure. 

In xerces-c-3.2.2/src/xercesc/util/Xerces_autoconf_config.hpp we want 
#define XERCES_HAVE_CSTDINT 0 for c++98
#define XERCES_HAVE_CSTDINT 1 for c++11

In xerces-c-3.2.2/config.h we want
All XERCES_USE_*_ICU commented out for c++98
All XERCES_USE_*_ICU defined out for c++11

### PR DESCRIPTION
**Pull request by gautierbureau**
And some cleaning of shell scripts to harmonize them. 
Fix #17

### CODE REVIEW

**Comment by gautierbureau** [comment]
Done

**Comment by rosiereflo** [comment]
envDynawo.sh is still mixing the 2 different developments.

**Comment by gautierbureau** [comment]
Done

File: dynawo/3rdParty/xerces-c/xerces-c-chain.sh [hunk 1]
```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
```
Comment (rosiereflo) [inline dynawo/3rdParty/xerces-c/xerces-c-chain.sh]: How come that this is not called in the toolchain file?

**Comment by rosiereflo** [comment]
Please make sure that libxml is using the Dynawo local xerces and not the default installation during compilation (cd FindXercesC.cmake)

File: dynawo/3rdParty/xerces-c/xerces-c-chain.sh (hunk 1 above)
Comment (rosiereflo) [reply dynawo/3rdParty/xerces-c/xerces-c-chain.sh]: WIll be added in another commit

**Pull request merged**